### PR TITLE
LUGG-834 Adding filter settings to center text in tables

### PR DESCRIPTION
**Pull request by angelamnr**
To test:

- On a clean build of luggage, pull down this branch.
-  Revert the luggage_ckeditor feature and clear cache.
-  Now, add a table. Add text to each cell and center that text with the CKEditor UI. Is it actually centered after you save?

### CODE REVIEW

File: luggage_ckeditor.features.filter.inc [hunk 1]
```diff
@@ -127,9 +127,10 @@ function luggage_ckeditor_filter_default_formats() {
         'settings' => array(
           'valid_elements' => 'a[!href|target|title],
 img[title|style|src|height|width|alt|align],
-table[align|width|height|border|cellpadding|cellspacing],tr,td[align|width|height|border|cellpadding|cellspacing|colspan|rowspan|class],th[align|width|height|border|cellpadding|cellspacing|colspan|rowspan|class],
+table[align|width|height|border|cellpadding|cellspacing|class],tr[rtcenter],td[align|width|height|border|cellpadding|cellspacing|colspan|rowspan|class],th[align|width|height|border|cellpadding|cellspacing|colspan|rowspan|class],
```
Comment (njbooher) [inline luggage_ckeditor.features.filter.inc]: Shouldn't this be 'class' for tr as well?

Comment (angelamnr) [reply luggage_ckeditor.features.filter.inc]: Yes, I think so. Good catch.

**Comment by jrearick** [comment]
LUGG-834 doesn't mention tables. LUGG-520 already has #19 merged should have fixed centering in tables in the td element.

**Comment by njbooher** [comment]
Why don't we just allow class on everything?

**Comment by angelamnr** [comment]
I guess I had tables on the brain because the site referenced in LUGG-834 used centered text almost exclusively for tables.

That's a good question. Would be pretty easy to allow class for everything.

**Comment by jrearick** [comment]
My initial thought was that we shouldn't add class to all elements because we want to go with the least permissive strategy. However, after thinking about it for a bit, class names are limited to the list given in the configuration and finding every little element that could appear and adding class when needed would be a game of whack-a-mole that we have been playing for a while now. I think the risk/benefit to allow class on all elements is favorable.

If we are allowing classes on everything, I thought it might be a good idea to check what classes CKEditor is providing. I found the following in luggage_ckeditor/ckeditor/css/ckeditor.css. We should allow them.

rteindent1
rteindent2
rteindent3
rteindent4
rteleft
rteright
rtecenter
rtejustify
ibimage_left
ibimage_right

**Comment by jrearick** [comment]
Accepting PR in #21 and closing this one as duplicate.

**Pull request merged**